### PR TITLE
Adds the ability to change description from the menu

### PR DESCRIPTION
**Pull request by SleepyMode**
https://github.com/NebulousCloud/helix/issues/18

### CODE REVIEW

File: gamemode/core/derma/cl_information.lua [hunk 1]
```diff
@@ -151,7 +156,9 @@ function PANEL:Init()
 				self.description:SetMouseInputEnabled(true)
 				self.description:SetCursor("hand")
 				self.description.DoClick = function(this)
-					RunConsoleCommand("ix", "CharDesc")
+					Derma_StringRequest(lDesc, lCmdCharDesc, self.description:GetText(), function(desc)
```
Comment (MrMe0w) [inline gamemode/core/derma/cl_information.lua]: Won't make any performance difference whatsoever if you just put L"phrase_name" directly here as an anonymous variable.

Comment (SleepyMode) [reply gamemode/core/derma/cl_information.lua]: I had issues with line length. I will just break it into multiple lines

Comment (alexgrist) [reply gamemode/core/derma/cl_information.lua]: It'll actually be more efficient in the function anyway because it's not needed until the string request.

**Comment by SleepyMode** [comment]
For whatever funny reason, it won't let me edit this, I'll open a new pull request.
I cannot even edit the commits.

**Pull request closed without merging**